src/main/java/com/splitspendings/groupexpensesbackend/model/SpendingComment.java

```diff
@@ -1,20 +1,28 @@
 package com.splitspendings.groupexpensesbackend.model;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import java.time.ZonedDateTime;
 
 @Entity
 @Table(name = "spending_comment")
-@Getter
-@Setter
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class SpendingComment {
 
     public static final int MESSAGE_MIN_LENGTH = 1;
     public static final int MESSAGE_MAX_LENGTH = 250;
 
+    public SpendingComment(String message, Spending spending, AppUser addedByAppUser) {
```

Comment:
I'd suggest not using constructors for creating data classes.  Data classes may have many many data fields, with different situations when different sets of parameters needed -> many constructors overloads, which IMHO makes code look messier. And in order for the whole code style to be consistent - not using constructors even in small data classes with few fields.
IMHO using setters/builders allows to have cleaner, more understandable code, even if it has more lines.